TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -505,8 +533,13 @@ public void DidChangeConfigurationParams(string[] arguments)
 
             //Configure CFG/DFA analyzer + external analyzers if any
             var compositeAnalyzerProvider = new CompositeAnalyzerProvider();
-            compositeAnalyzerProvider.AddActivator((o, t) => CfgDfaAnalyzerFactory.CreateCfgAnalyzer(TypeCobolLanguageServer.lspcfgId, Configuration.CfgBuildingMode));
-            compositeAnalyzerProvider.AddCustomProviders(Configuration.CustomAnalyzerFiles);
+            compositeAnalyzerProvider.AddActivator((o, t) => CfgDfaAnalyzerFactory.CreateCfgAnalyzer(TypeCobolLanguageServer.lspcfgId, Configuration.CfgBuildingMode));            
+            if (this._customAnalyzerProviders != null)
```

Comment:
Same here, the array of analyzers is non-null, so an Assert is better.

Reply:
don in 5ec844a